lib/msf/core/modules/loader/executable.rb

```diff
@@ -88,6 +88,22 @@ def read_module_content(parent_path, type, module_reference_name)
         elog "Unable to load module #{full_path}, unknown module type"
         return ''
       end
+    rescue Errno::ENOENT => e
```

Comment:
Catching this exception here seems like a leaky abstraction; Maybe we should be making changes to how LoadError exceptions are raised/caught? 🤔 

https://github.com/rapid7/metasploit-framework/blob/eefe174e3a4d953877c1cf5b5f194b21e17017b3/lib/msf/core/modules/external/shim.rb#L8-L10